pragma-ui/src/app/assets/_components/asset-row.tsx

```diff
@@ -0,0 +1,218 @@
+    return Number(asset.variations.past1h) > 0
+      ? "▲"
+      : asset.variations.past1h === 0
+      ? "-"
+      : "▼";
+  };
+
+  return (
+    <Link
+      href={
+        !hasError
+          ? `/asset/${encodeURIComponent(asset.ticker).replace("%2F", "-")}`
+          : hasError
+          ? "#"
+          : `/provider/${asset.ticker}`
+      }
+      scroll={false}
+      className={classNames(
+        styles.assetPerf,
+        hasError && "opacity-70",
+        isApiSource && styles.assetPerfApi
+      )}
+      onClick={(e) => {
+        if (hasError) {
+          e.preventDefault();
+        }
+      }}
+    >
+      <div className="my-auto flex flex-row gap-4 text-LightGreenFooter md:tracking-wider">
+        {loading ? (
+          <div className="my-auto h-8 w-8 animate-pulse rounded-full bg-lightBlur" />
+        ) : asset.image && !imageError ? (
+          <Image
+            height={30}
+            width={30}
+            alt="AssetImage"
+            src={asset.image}
+            onError={() => setImageError(true)}
+          />
+        ) : (
+          <div className="my-auto size-8 rounded-full bg-lightBlur text-center inline-flex justify-center">
+            <span className="self-center">{asset.ticker[0]}</span>
+          </div>
+        )}
+        {loading ? (
+          <div className="flex flex-col text-lg text-lightGreen">
+            <div className="my-auto mb-1 h-3 w-14 animate-pulse rounded-full bg-lightBlur"></div>{" "}
+            <div className="my-auto h-2 w-8 animate-pulse rounded-full bg-lightBlur"></div>
+          </div>
+        ) : (
+          <div className="text-md flex flex-col text-lightGreen">
+            {asset.ticker}{" "}
+            <div className="font-mono text-xs uppercase text-LightGreenFooter md:tracking-wider">
+              {asset.type}
+            </div>
+          </div>
+        )}
+      </div>
+      <div className="my-auto flex flex-row gap-2 font-mono text-xs md:tracking-wider">
+        {loading ? (
+          <div className=" my-auto h-3 w-24 animate-pulse rounded-full bg-lightBlur"></div>
+        ) : (
+          <div className={isUnsupported ? "text-redDown" : "text-lightGreen"}>
+            {asset.lastUpdated}
+          </div>
+        )}
+      </div>
+      <div className="my-auto flex flex-row gap-2 font-mono text-sm text-lightGreen md:tracking-wider">
+        {loading ? (
+          <div className="my-auto h-3 w-14 animate-pulse rounded-full bg-lightBlur"></div>
+        ) : (
+          <div>{asset.sources}</div>
+        )}
+      </div>
+      <div className="my-auto flex flex-row gap-2 font-mono text-sm md:tracking-wider">
+        {loading ? (
+          <div className="my-auto h-3 w-20 animate-pulse rounded-full bg-lightBlur"></div>
+        ) : isUnsupported ? (
+          <div className="text-redDown">N/A</div>
+        ) : (
+          <div className={priceChangeClass}>
+            ${Number.parseFloat(String(asset.price)).toFixed(2)}
```

Comment:
```suggestion
            ${Number.parseFloat(String(asset.price)).toFixed(3)}
```

make sure this doesnt fck up rendering, but we need at least 3 decimals of precision if not more cause some shitcoins are like 0.00001$ you know